rpc/rpc-wrappers.go

```diff
@@ -664,21 +664,21 @@ func (vac ValidationAuthorityClient) PerformValidation(ctx context.Context, doma
 
 // IsSafeDomain returns true if the domain given is determined to be safe by an
 // third-party safe browsing API.
-func (vac ValidationAuthorityClient) IsSafeDomain(ctx context.Context, req *core.IsSafeDomainRequest) (*core.IsSafeDomainResponse, error) {
-	data, err := json.Marshal(req)
+func (vac ValidationAuthorityClient) IsSafeDomain(ctx context.Context, domain string) (isSafe bool, err error) {
```

Comment:
I should have mentioned this earlier: the reason we have the seemingly-silly `IsSafeDomainRequest` and `IsSafeDomainResponse` types was that we were trying to arrange things to be easy for the gRPC transition. The idea was that we would replace those structs with the generated PBs. So, rather than change this function to take a string, I would keep it as taking an `IsSafeDomainRequest` but use the protobuf one instead of the struct, and delete the struct.